client/lib/plans/constants.js

```diff
@@ -983,15 +1042,19 @@ export const FEATURES_LIST = {
 	},
 	[ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY ]: {
 		getSlug: () => FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
-		getTitle: () => i18n.translate( 'Daily Offsite Backups' ),
+		getTitle: () => i18n.translate( 'Daily Off-site Backups' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Daily off-site backups' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-plans-copy-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).